packages/shoreline/src/themes/sunrise/components/table.css

```diff
@@ -6,20 +6,6 @@
 [data-sl-table-row] {
   display: contents;
   color: var(--sl-fg-base);
-
-  &[data-selected='true'] {
-    &[data-sl-table-cell] {
-      background: var(--sl-bg-accent);
-    }
-  }
-
-  &[data-dim-on-hover='true']:hover {
-    &[data-selected='false'] {
-      &[data-sl-table-cell] {
-        background: var(--sl-bg-base-soft);
-      }
-    }
-  }
 }
```

Comment:
I thought the fix would be as simple as changing those two lines. Why did you have to change all those things?
```css
&[data-selected='true'] {
    & > [data-sl-table-cell] {
      background: var(--sl-bg-accent);
    }
  }

  &[data-dim-on-hover='true']:hover {
    &[data-selected='false'] {
      & > [data-sl-table-cell] {
        background: var(--sl-bg-base-soft);
      }
    }
  }
```

Reply:
Da erro de lint
Por isso tive que mudar